pyproject.toml

```diff
@@ -15,6 +15,7 @@ pyreadline3 = "^3.4.1"
 colorama = "^0.4.6"
 urllib3 = "^1.26.15"
 typing-extensions = "^4.9.0"
+packaging = "24.0"
```

Comment:
Cool. It just worked for me, so I thought maybe it was a builtin, but I guess not.

Reply:
Yup. poetry.lock change seems to indicate it's a transitive dependency. So a poetry env would probably just have it.